features/ptp/deploy/03_subscription.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ptp-operator-subsription
+  namespace: openshift-ptp
+spec:
+  channel: "${channel}"
+  name: ptp-operator
+  source: opsrctest
```

Comment:
I think this can be a variable as when it is released it won't be in the opsrctest but in the Red Hat operators

Reply:
nice catch thanks!